docs/source/using-pbft-consensus.rst

```diff
@@ -21,6 +21,14 @@ restricted membership. It has the following requirements:
   using the on-chain settings ``sawtooth.consensus.algorithm.name`` and
   ``sawtooth.consensus.algorithm.version``.
 
+  - The PBFT consensus engine name is ``pbft``.
```

Comment:
Your commit message still says that the name is in the Cargo.toml file please updated.

Reply:
Fixed.